lib/ecto/repo.ex

```diff
@@ -975,6 +975,7 @@ defmodule Ecto.Repo do
 
       # Use a keyword list to preload nested associations as well
       posts = Repo.preload posts, [comments: [:replies, :likes], authors: []]
+      posts = Repo.preload posts, [comments: [replies: [:reactions], :likes], :authors]
```

Comment:
```suggestion

      # You can mix atoms and keywords, but the atoms must come first
      posts = Repo.preload posts, [comments: [:likes, replies: [:reactions]], :authors]
```

Reply:
Thanks @josevalim , I was about to fix the order for that right before the merge. But wouldn't that affect the `:authors` atom as well?

Something like:

```elixir
posts = Repo.preload posts, [:authors, comments: [:likes, replies: :reactions]]
```